source/PlayerInfo.cpp

```diff
@@ -3663,6 +3663,17 @@ void PlayerInfo::RegisterDerivedConditions()
 	visitedSystemProvider.SetGetFunction(visitedSystemFun);
 	visitedSystemProvider.SetHasFunction(visitedSystemFun);
 
+	auto &&pluginProvider = conditions.GetProviderPrefixed("installed plugin: ");
+	auto pluginFun = [this](const string &name) -> bool
```

Comment:
```suggestion
	auto pluginFun = [](const string &name) -> bool
```
we dont need the this here